### PR TITLE
add storing prediction/qrel files option for test set

### PR DESCRIPTION
**Pull request by Victor0118**
@tuzhucheng Could you review this?

Thanks!

### CODE REVIEW

File: utils/relevancy_metrics.py [hunk 1]
```diff
@@ -30,7 +30,11 @@ def get_map_mrr(qids, predictions, labels, device=0):
     mean_average_precision = float(trec_out_lines[0].split('\t')[-1])
     mean_reciprocal_rank = float(trec_out_lines[1].split('\t')[-1])
 
-    os.remove(qrel_fname)
-    os.remove(results_fname)
+    if keep_results:
+        print("saving prediction file to {}".format(results_fname))
```
Comment (tuzhucheng) [inline utils/relevancy_metrics.py]: Please capitalize error message

File: mp_cnn/__main__.py [hunk 2]
```diff
@@ -114,9 +118,12 @@ def evaluate_dataset(split_name, dataset_cls, model, embedding, loader, batch_si
     else:
         raise ValueError('optimizer not recognized: it should be either adam or sgd')
 
-    train_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, train_loader, args.batch_size, args.device)
-    test_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, test_loader, args.batch_size, args.device)
-    dev_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, dev_loader, args.batch_size, args.device)
+    train_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, train_loader, args.batch_size,
```
Comment (tuzhucheng) [inline mp_cnn/__main__.py]: If the help message is "store the output score and qrel files into disk for the test set" why is it added for training set as well?

Comment (Victor0118) [reply mp_cnn/__main__.py]: you are right. I will delete them.

File: utils/relevancy_metrics.py (hunk 1 above)
Comment (Victor0118) [reply utils/relevancy_metrics.py]: Thx!
